modules/social_features/social_footer/social_footer.permissions.yml

```diff
@@ -0,0 +1,3 @@
+administer social_footer settings:
```

Comment:
If we're adding this can we also change the routing.yml file?

https://github.com/goalgorilla/open_social/blob/8ca7907fceb1c542f36b91a8d64bf1c6e2b8fb83/modules/social_features/social_footer/social_footer.routing.yml#L7

The permission is called `administer social_footer_block settings` there.

Reply:
@bramtenhove I have corrected it :-)